gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Feb 25 19:14:47 IST 2020
+#Mon Jan 04 20:46:13 IST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```

Comment:
Revert this to the older version for now

Reply:
Gradle Version 6.1.1 and Gradle Plugin Version 4.0.1 are required for ViewBinding